### PR TITLE
chore(node/rolldown): import type.

### PR DESCRIPTION
**Pull request by Cecil0o0**

### CODE REVIEW

File: packages/rolldown/src/cli/utils.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { pathToFileURL } from 'node:url'
 import nodePath from 'node:path'
 import { createConsola } from 'consola'
-import { RolldownConfigExport } from '../types/rolldown-config-export.js'
+import type { RolldownConfigExport } from '../types/rolldown-config-export.js'
```
Comment (IWANABETHATGUY) [inline packages/rolldown/src/cli/utils.ts]: Is `import type` from a js module meaningful?

**Comment by IWANABETHATGUY** [comment]
Can you split the PR into two parts?
one for solving the circular dependency, and another for changing `import` to `import type`?

**Comment by IWANABETHATGUY** [comment]
Could you share the tools that check **circular dependency**, so I could verify if there are more **circular dependency** in our codebase?

File: packages/rolldown/src/cli/utils.ts (hunk 1 above)
Comment (Cecil0o0) [reply packages/rolldown/src/cli/utils.ts]: It's not. At that time, I didn't see this module specifier which ends with `.js`, and now I find this module actually is a ts module, aha.

would you think remove suffix, be a appropriate way for making meaningful?

**Comment by Cecil0o0** [comment]
> Can you split the PR into two parts? one for solving the circular dependency, and another for changing `import` to `import type`?

of course could, I'd like to do this.

**Comment by Cecil0o0** [comment]
> Could you share the tools that check **circular dependency**, so I could verify if there are more **circular dependency** in our codebase?

I would try to do this in another PR. Maybe a change serves as a supplement, because rolldown has done it natively, `packages/rolldown` had been checked natively by rolldown ourself. #1231

**Review by IWANABETHATGUY — APPROVED** [review]

**Pull request merged**